### ISSUE TITLE
Can you share the loss curve of the  Swin-UNETR pre-training process

### ISSUE THREAD

**Issue by upupming** [Project-MONAI/research-contributions#73]
Hi, Thanks for your great work on Swin-UNETR, I am trying to run pre-training on another dataset (~2000 CTs). But the loss curve seems not to be decreased:

<img width="761" alt="image" src="https://user-images.githubusercontent.com/24741764/176806185-7ddecfbc-578a-4b7a-9502-60927d23a419.png">

Could you share your loss curve on the 5050 CTs dataset? Thank you very much!

I am pre-training the model on a single GPU with batch size 2.

**Comment by ahatamiz** [comment]
Hi @upupming

I see. We trained the model using 4 nodes with 8 GPUs (total 32). We noticed scaling up the pre-training to multi-node is important for convergence.

**Comment by GaoHuaZhang** [comment]
Hi @ahatamiz, i meet the same problem ,as i pre-training the model on a single GPU with batch size 2.My input and ground-turth are shown in the image, but the output does not seem to have any resemblance to the original image, do you mean to train on multiple GPUs to have good results
![x1_aug](https://user-images.githubusercontent.com/73919261/201830691-eefe2fd2-2525-4df7-abbc-0feba7c733c8.png)
x1_aug
![x1_gt](https://user-images.githubusercontent.com/73919261/201830693-f09afee7-e284-4b2c-9d75-720d1310caf0.png)
x1_gt
![x1_recon](https://user-images.githubusercontent.com/73919261/201830740-5e9af51c-9a2d-48ce-bd7c-1b3c902b2ed6.png)
x1_recon

**Comment by tangy5** [comment]
hI @GaoHuaZhang ,

Thanks, batch size should be a key point. We don't have the record for this training any more, but we have this
https://github.com/Project-MONAI/tutorials/tree/main/self_supervised_pretraining
which is a similar strategy pre-trianing, you could see loss curves with this tutorial.

Thanks

**Comment by lyangfan** [comment]
> Hi @ahatamiz, i meet the same problem ,as i pre-training the model on a single GPU with batch size 2.My input and ground-turth are shown in the image, but the output does not seem to have any resemblance to the original image, do you mean to train on multiple GPUs to have good results ![x1_aug](https://user-images.githubusercontent.com/73919261/201830691-eefe2fd2-2525-4df7-abbc-0feba7c733c8.png) x1_aug ![x1_gt](https://user-images.githubusercontent.com/73919261/201830693-f09afee7-e284-4b2c-9d75-720d1310caf0.png) x1_gt ![x1_recon](https://user-images.githubusercontent.com/73919261/201830740-5e9af51c-9a2d-48ce-bd7c-1b3c902b2ed6.png) x1_recon

Have you solved the problem? I have the same problem as you

**Comment by verse-armour** [comment]
> 你好[@ahatamiz](https://github.com/ahatamiz)，我遇到了同样的问题，因为我在批量大小为 2 的单个 GPU 上对模型进行预训练。我的输入和 ground-turth 显示在图像中，但输出似乎与原始图像没有任何相似之处，你的意思是在多个 GPU 上训练以获得良好的结果 x1_aug x1_gt x1_recon ![x1_aug](https://user-images.githubusercontent.com/73919261/201830691-eefe2fd2-2525-4df7-abbc-0feba7c733c8.png) ![x1_gt](https://user-images.githubusercontent.com/73919261/201830693-f09afee7-e284-4b2c-9d75-720d1310caf0.png) ![x1_recon](https://user-images.githubusercontent.com/73919261/201830740-5e9af51c-9a2d-48ce-bd7c-1b3c902b2ed6.png)



> Hi, Thanks for your great work on Swin-UNETR, I am trying to run pre-training on another dataset (~2000 CTs). But the loss curve seems not to be decreased:
> 
> <img alt="image" width="761" src="https://user-images.githubusercontent.com/24741764/176806185-7ddecfbc-578a-4b7a-9502-60927d23a419.png">
> 
> Could you share your loss curve on the 5050 CTs dataset? Thank you very much!
> 
> I am pre-training the model on a single GPU with batch size 2.

hi, I face a similar situation. The loss curve is oscillation. Have you solved the problem?

<img width="1129" height="547" alt="Image" src="https://github.com/user-attachments/assets/2d926807-29b0-4098-b93d-65c90e0286d8" />

<img width="1055" height="713" alt="Image" src="https://github.com/user-attachments/assets/0c8cdea2-f2b6-4641-861c-508a6faa92b3" />
